tests/graphql_mock/mod.rs

```diff
@@ -3,18 +3,24 @@ use std::fs;
 use regex::Regex;
 use serde_json::Value;
 
+#[must_use]
 pub fn start_mock_server() -> mockito::Server {
```

Comment:
As discussed on discord, unless there is something specific that we can't test via http_spec, let's not add another way to write a test. It becomes quite confusing for contributors to write tests when there are multiple ways to do the same thing.

Reply:
removed this and test that used it. 

put my thoughts on http_spec to https://github.com/tailcallhq/tailcall/issues/702